outputs.tf

```
@@ -43,6 +43,21 @@ output "nat_instance_ids" {
   value       = aws_instance.nat_instance.*.id
 }
 
+output "nat_instance_security_group_id" {
+  value       = module.nat_instance_security_group.id
+  description = "NAT Instance Security Group ID"
+}
+
+output "nat_instance_security_group_arn" {
+  value       = module.nat_instance_security_group.arn
+  description = "NAT Instance Security Group ARN"
+}
+
+output "nat_instance_security_group_name" {
+  value       = module.nat_instance_security_group.name
+  description = "NAT Instance Security Group name"
+}
```

```suggestion
output "nat_instance_security_group" {
  value       = module.nat_instance_security_group
  description = "NAT Instance Security Group"
}
```

Why not just output the entire module ? This way we don't have to update the outputs if there is ever an additional output from the security group module.